src/index.tsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import Root from "./App";
 import reportWebVitals from "./reportWebVitals";
-import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import { RouterProvider, createHashRouter } from "react-router-dom";
```

Comment:
The welcome and app test files should be updated too for consistency across the app